### PR TITLE
Reworded the explanation about the kernel.event_listener tag

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | #4680 |

### CODE REVIEW

File: reference/dic_tags.rst [hunk 1]
```diff
@@ -540,14 +540,16 @@ kernel.event_listener
 
 **Purpose**: To listen to different events/hooks in Symfony
 
-This tag allows you to hook your own classes into Symfony's process at different
-points.
+During the execution of a Symfony application, different events are triggered.
+This tag allows you to *hook* your own classes into any of those events.
```
Comment (xabbuh) [inline reference/dic_tags.rst]: We should add something that explains that you can also trigger your own events that will be dispatched through the same service as the core ones.

Comment (javiereguiluz) [inline reference/dic_tags.rst]: I like your proposal but I'm stuck trying to implement it. Do you have some suggestion for the new explanation? Thanks!

Comment (xabbuh) [inline reference/dic_tags.rst]: I suggest to use something like this:

> During the execution of a Symfony application, different events are triggered and you can also dispatch custom events. This tag allows [...]

**Comment by javiereguiluz** [comment]
Reworded and rebase. Thanks @xabbuh for your help.

**Comment by WouterJ** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks guys!

**Pull request merged**